charts/acp/templates/ingress.yaml

```diff
@@ -86,20 +91,25 @@ metadata:
     {{- end }}
     nginx.ingress.kubernetes.io/auth-tls-verify-client: "optional_no_ca"
     nginx.ingress.kubernetes.io/auth-tls-verify-depth: "2"
-    nginx.ingress.kubernetes.io/auth-tls-secret: {{ include "acp.mtls.secrets" . }}
+    nginx.ingress.kubernetes.io/auth-tls-secret: "{{ include "acp.mtls.secrets" . }}"
     {{- if .Values.tlsDisabled }}
     nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
     {{- else }}
     nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     {{- end }}
     nginx.ingress.kubernetes.io/proxy-ssl-ciphers: "EECDH+AESGCM:EDH+AESGCM"
     nginx.ingress.kubernetes.io/proxy-ssl-protocols: "TLSv1.2"
+    nginx.ingress.kubernetes.io/proxy-ssl-secret: "{{ .Release.Namespace }}/{{ include "acp.fullname" . }}-tls"
+    nginx.ingress.kubernetes.io/proxy-ssl-server-name: "on"
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
-    nginx.ingress.kubernetes.io/ssl-redirect: "true"
     nginx.ingress.kubernetes.io/proxy-buffer-size: "8k"
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
     nginx.ingress.kubernetes.io/ssl-ciphers: "EECDH+AESGCM:EDH+AESGCM"
     nginx.ingress.kubernetes.io/configuration-snippet: |
       proxy_set_header X-SSL-CERT $ssl_client_escaped_cert;
+    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
```

Comment:
Can we keep it configurable? Same with owasp.

Reply:
updated